src/TusServer.php

```diff
@@ -356,18 +351,13 @@ protected function handlePatch(ServerRequestInterface $request, ResponseInterfac
                 ) {
                     throw new RuntimeException('Error when copying ' . $chunkFile->getPathname() . ' to target file ' . $targetFile->getPathname());
                 }
-            } catch (RuntimeException $t) {
```

Comment:
This resolves a phpstan issue. Also, by just omitting the `catch` the `RuntimeException` is thrown anyway.